stm32-modules/heater-shaker/firmware/host_comms_task/freertos_comms_task.cpp

```diff
@@ -27,10 +28,12 @@ struct CommsTaskFreeRTOS {
     USBD_CDC_ItfTypeDef cdc_class_fops;
     USBD_HandleTypeDef usb_handle;
     USBD_CDC_LineCodingTypeDef linecoding;
-
+    UART_HandleTypeDef uart_handle;
```

Comment:
this struct is starting to be a lot; maybe could we make a `struct CommsTaskUSBData` and `struct CommsTaskUARTData` and then have `struct CommsTaskFreeRTOS { CommsTaskUSBData usb; CommsTaskUARTData uart; };`?